DOCS/man/options.rst

```diff
@@ -3937,6 +3937,13 @@ Demuxer
     libarchive opens all volumes anyway when playing the main file, even though
     mpv iterated no archive entries yet.
 
+``--demuxer-playlist-recursive-dir=<yes|no>``
```

Comment:
Just had a thought, but how about renaming this to `recursive-dir` for simplicity?

Reply:
We could do that, as the name is rather long, but the other demuxer specific options also have `demuxer-<demuxer name>-<option>`.

Reply:
`sub-codepage` and `rtsp-transport` in demux_lavf don't, so there's precedent.